src/Model/TalkCommentMapper.php

```diff
@@ -453,6 +503,7 @@ public function getRawComment($comment_id)
         if ($response) {
             $results = $stmt->fetchAll(PDO::FETCH_ASSOC);
             if (isset($results[0])) {
+                $results[0]['created_at'] = new \DateTimeImmutable($results[0]['created_at']);
```

Comment:
This should be `new DateTimeImmutable($results[0]['created_at'], new DateTimeZone('UTC'))`